python/tests/test_table_read.py

```diff
@@ -493,6 +493,22 @@ def test_writer_fails_on_protocol():
         dt.to_pandas()
 
 
+@pytest.mark.parametrize("version, expected", [(2, (5, 3))])
+def test_peek_next_commit(version, expected):
+    table_path = "../crates/deltalake-core/tests/data/simple_table"
+    dt = DeltaTable(table_path)
+    actions, current_version = dt.peek_next_commit(version=version)
+    assert (len(actions), current_version) == expected
+
+
+def test_delta_log_not_found():
+    table_path = "../crates/deltalake-core/tests/data/simple_table"
+    dt = DeltaTable(table_path)
+    latest_version = dt._table.get_latest_version()
```

Comment:
Stylistically, this is access to a private member `_table`. Is that OK or do we need to promote `get_latest_version()` to also be a public member function of `DeltaTable`?

Reply:
Added `get_latest_version()` as a public member.